readme.md

```diff
@@ -30,12 +30,27 @@
 
 ### Open Source
 
+#### Casual
+
+ - [**Bob Box**](https://github.com/fireslime/bounce_box) - Bob Box is a simple casual game writen in Flutter+Flame. By [FireSlime](https://fireslime.xyz)
+ - [Add Your Item](https://github.com/flame-engine/awesome-flame/pulls)
+
+#### Endless Runner
+
+ - [**BGUG**](https://github.com/fireslime/bgug) - Break Guns Using Gems is a fast paced side-scrolling platformer with innovative controls and and a gun-related twist. By [FireSlime](https://fireslime.xyz)
```

Comment:
We need to have a common standard. I am okay with using `. By X` in favor of `by X` but then we need to adjust contributing.md

Reply:
I can change to `by X`

Reply:
actually I think `. By X` or `, by X` are better, what do you think?

Reply:
Let us do `. By X` then :)